src/server/server.js

```diff
@@ -38,6 +38,16 @@ app.get("/stock/:nse", (req, res) => {
   });
 });
 
+app.get("/stockChart/:params", (req, res) => {
+  const {params} = encodeURIComponent(req.params);
```

Comment:
Thanks!
However; I still can't even get here, as I keep getting a 500 that states that getStockChartUrl() is not defined :/